_security-plugin/multi-tenancy/mt-agg-view.md

```diff
@@ -0,0 +1,74 @@
+{: .note }
+
+### Feature benefits
+
+- Implementing an aggregate view of all saved objects on one screen allows a user to quickly locate an object of interest and determine which tenant is associated with it. Once the user locates an object, the user can select the appropriate tenant and work with the object.
+- This feature also adds a Tenants column to the Saved Objects table, which allows users to filter the view by tenants and their associated saved objects.
+
+### Plans for future development
+
+In subsequent releases, we plan to expand on the functionality of this feature to include the ability to perform actions directly from aggregate view and share items without having to first select a specific tenant. In the longer term, OpenSearch plans to evolve multi-tenancy so that it becomes a much more flexible tool for sharing objects among users and easier to do so by a more sophisticated way of assigning the roles and permissions that faciliate sharing. To learn more about the features being proposed for future releases, see the GitHub issue [Dashboards object sharing](https://github.com/opensearch-project/security/issues/1869).
+
+### Known limitations
+
+In this first experimental phase of development, there are some limitations that should be observed before enabling the feature and using it in a test environment.
+
+* The feature can only be used in a new cluster. At this time, the feature is not suported by clusters already in use.
+* Also, the feature should be used only in a test environment, not in production. 
+* Finally, once the feature has been enabled and used in a test cluster, the feature cannot be disabled for the cluster. Disabling the feature once it has been used to work with tenants and saved objects can result in the loss of saved objects and have an impact on tenant-to-tenant functionality. 
+
+These limitations will be addressed in upcoming releases.
+
+## Enabling aggregate view for saved objects
+
+By default, the aggregate view in the Saved Objects table is disabled. To enable the feature, add the `opensearch_security.multitenancy.enable_aggregation_view` flag to the `opensearch_dashboards.yml` and set it to `true`.
+
+`opensearch_security.multitenancy.enable_aggregation_view: true`
+
+After enabling the feature you can start the new cluster and then launch Dashboards. 
+
+## Working in aggregate view
+
+Select the drop-down arrow on the Tenants column to display the list of tenants available to the user. You can select multiple tenants while the menu is open.
+
+<img src="{{site.url}}{{site.baseurl}}/images/Security/tenant_column.png" alt="Dashboards Saved Objects view with emphasis on Tenants column" width="700">
+
+Select anywhere outside the menu to collapse it. The Title column displays the saved objects corresponding to the selected tenants. The Tenants column displays the tenants associated with the saved objects. Also, the number of tenants selected for filtering is expressed in a red box on the drop-down.
```

Comment:
>The Title column displays the saved objects corresponding to the selected tenants.

This might need to be changed. The aggregation view displays the saved objects in all permitted tenants instead.

Reply:
I rewrote content around this as well as this sentence to clear things up. Good call.